R/scale-type.R

```diff
@@ -25,7 +24,13 @@ find_scale <- function(aes, x, env = parent.frame()) {
 # Look for object first in parent environment and if not found, then in
 # ggplot2 namespace environment.  This makes it possible to override default
 # scales by setting them in the parent environment.
-find_global <- function(name, env, mode = "any") {
+find_global <- function(name, env, mode = "any", theme = ggplot2::theme()) {
```

Comment:
Thinking some more about this, I don't understand the point of `theme = ggplot2::theme()`. Just write `theme = NULL`.